integration-tests/fixtures/generate-bundle-stats/solidstart/solidstart-plugin.test.ts

```diff
@@ -0,0 +1,202 @@
+                name: expect.any(String),
+                size: expect.any(Number),
+                chunkUniqueIds: expect.arrayContaining([expect.any(String)]),
+              },
+            ]),
+          });
+
+          // fetch stats from the server
+          const serverRes = await fetch(
+            `http://localhost:8000/get-stats-by-bundle-name/${id}/${serverBundleName}`,
+          );
+          const serverData = (await serverRes.json()) as { stats: string };
+          const serverStats = JSON.parse(serverData.stats) as unknown;
+
+          expect(serverStats).toMatchSnapshot({
+            version: expect.any(String),
+            builtAt: expect.any(Number),
+            duration: expect.any(Number),
+            outputPath: expect.any(String),
+            bundleName: expect.stringContaining(serverBundleName),
+            plugin: {
+              name: expect.stringMatching("@codecov/solidstart-plugin"),
+              version: expect.any(String),
+            },
+            bundler: {
+              name: expect.any(String),
+              version: expect.any(String),
+            },
+            assets: expect.arrayContaining([
+              {
+                name: expect.any(String),
+                normalized: expect.any(String),
+                size: expect.any(Number),
+                gzipSize: expect.anything(),
+              },
+            ]),
+            chunks: expect.arrayContaining([
+              {
+                id: expect.any(String),
+                uniqueId: expect.any(String),
+                initial: expect.any(Boolean),
+                entry: expect.any(Boolean),
+                files: expect.arrayContaining([expect.any(String)]),
+                names: expect.arrayContaining([expect.any(String)]),
+              },
+            ]),
+            modules: expect.arrayContaining([
+              {
+                name: expect.any(String),
+                size: expect.any(Number),
+                chunkUniqueIds: expect.arrayContaining([expect.any(String)]),
+              },
+            ]),
+          });
+        },
+        { timeout: 25_000 },
+      );
+    });
+
+    describe("invalid bundle name is passed", () => {
+      beforeEach(async () => {
+        const config = new GenerateConfig({
+          plugin: "solidstart",
+          configFileName: "app",
+          format: "esm",
+          detectFormat: "esm",
+          version: `v1`,
+          detectVersion: "v1",
+          file_format: "ts",
+          enableSourceMaps: false,
+          overrideOutputPath: `${solidStartApp}/app.config.ts`,
+        });
+
+        await config.createConfig();
+        config.removeBundleName(`test-solidstart-v${version}`);
+        await config.writeConfig();
+      });
+
+      afterEach(async () => {
+        await $`rm -rf ${solidStartApp}/app.config.ts`;
+        await $`rm -rf ${solidStartApp}/.solid-start`;
+        await $`rm -rf ${solidStartApp}/app.config.ts.timestamp-*`;
```

Comment:
Looks like this needs to be updated to match the ones you have committed from this directory.